src/CommunityToolkit.HighPerformance/Box{T}.cs

```diff
@@ -218,4 +218,35 @@ public static class BoxExtensions
         // this doesn't really matter in this case anyway.
         return ref Unsafe.Unbox<T>(box);
     }
+
+    /// <summary>
+    /// Gets a <typeparamref name="T"/> value from a <see cref="Box{T}"/> instance.
+    /// </summary>
+    /// <typeparam name="T">The type of value to retrieve.</typeparam>
+    /// <param name="box">The input <see cref="Box{T}"/> instance.</param>
+    /// <returns>The <typeparamref name="T"/> boxed value within <paramref name="box"/>.</returns>
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    public static T GetValue<T>(this Box<T> box)
```

Comment:
As a side point to the fact we're not planning on adding any new APIs to the `Box<T>` type, keep in mind that this method is also not really doing anything anyway, as there's already the implicit operator that's doing this. So this would really just be a slightly different syntax for doing what is already possible via just implicit conversion (or cast).